data/schema.graphql

```diff
@@ -776,7 +776,7 @@ type Artwork implements Node & Sellable {
   # Returns an HTML string representing the embedded content (video)
   embed(width: Int = 853, height: Int = 450, autoplay: Boolean = false): String
   edition_of: String
-  edition_sets: [EditionSet]
+  edition_sets(sort: EditionSetSorts): [EditionSet]
```

Comment:
**Note to reviewers:** I haven't made any changes that should have caused this change to the schema, but I committed it because GraphQL works in mysterious ways. Please let me know if this seems strange.